CTP/ha_repl/src/com/navercorp/cubridqa/ha_repl/impl/FeedbackFile.java

```diff
@@ -109,7 +114,8 @@ public void onTestCaseStopEvent(String testCase, boolean flag, long elapseTime,
 		} else {
 			head = "[UNKNOWN]";
 		}
-
+		
+		this.totalExecutedCaseNum = totalSuccNum + totalFailNum;
```

Comment:
as is:
this.totalCaseNum = this.totalExecutedCaseNum + this.totalSkipNum;
to be:
move to updateTestingStatistics().